components/scream/src/dynamics/homme/atmosphere_dynamics_rayleigh_friction.cpp

```diff
@@ -0,0 +1,93 @@
+#include "atmosphere_dynamics.hpp"
+
+// Scream includes
+#include "share/util/scream_common_physics_functions.hpp"
+
+// Homme includes
+#include "dynamics/homme/homme_dimensions.hpp"
+
+namespace scream {
+
+void HommeDynamics::rayleigh_friction_init()
+{
+  constexpr int N = HOMMEXX_PACK_SIZE;
+  using KT = KokkosTypes<DefaultDevice>;
+  using Pack = ekat::Pack<Real,N>;
+
+  const int nlevs = m_dyn_grid->get_num_vertical_levels();
+  const auto npacks= ekat::PackInfo<Pack::n>::num_packs(nlevs);
+  m_otau = decltype(m_otau)("otau", npacks);
```

Comment:
If m_raytau == 0, you don't need to allocate this view. Not a big deal since it's a small view, but might as well skip the allocation.